lib/InternalContactsFieldArray/UserField.js

```diff
@@ -49,18 +56,40 @@ export default class UserField extends React.Component {
       selectUser={this.props.onUserSelected}
       renderTrigger={(props) => {
         this.triggerButton = props.buttonRef;
+
+        const userString = this.props.user.personal ? `${this.props.user.personal.lastName}${this.props.user.personal.firstName ? ', ' : ' '}${this.props.user.personal.firstName ? this.props.user.personal.firstName : ''} ${this.props.user.personal.middleName ? this.props.user.personal.middleName : ''}` : `user ${this.props.index + 1}`;
```

Comment:
We've got a bunch of these statements littered throughout the library now. Ideally we'd just define this in one place and import it everywhere we need to get a user's name as a string. 

I'll make a pass through after this is merged and clean it up.